### ISSUE TITLE
Tags or releases

### ISSUE THREAD

**Issue by Grimler91** [tcgoetz/utilities#5]
Hi!

I am looking at creating a set of PKGBUILD scripts for [archlinux's user repository](https://aur.archlinux.org) for python-idbutils to make it easier for myself and others to install GarminDB. 

The latest idbutils release on pypi is 1.1.0, which I guess corresponds to [this commit](https://github.com/tcgoetz/utilities/commit/97dd3db4aad7bb63e93a499c2d3cec738823638b), but there is no corresponding tag or release in this repository. Would it be possible to create a tag or release in this repository for next version, to simplify packaging?

**Comment by tcgoetz** [comment]
yes, I should make releases in the GarminDb subrepos. I've just been lazy. I'll try to be better.
